packages/blinks-core/src/api/Action/Action.ts

```diff
@@ -174,10 +174,6 @@ export class Action {
     return this._adapter;
   }
 
-  public setAdapter(adapter: ActionAdapter) {
```

Comment:
I'd leave this method for backwards compatibility

Reply:
hm, but this one actually doesn't fully functional and therefore not recommended to use
may be remove?

Reply:
it is functional though. If you don't need to use it in react for instance